recipes/koalas/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "koalas" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ed16834e255924ddfe9a254cbb258b35dd167b727f223bee316ee436778794d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<35]
```

This looks like it could be `noarch`?
```suggestion
  noarch: python
```